Hazelcast.Net/Hazelcast.Client.Spi/Exception.cs

```diff
@@ -16,41 +16,58 @@
 using Hazelcast.Core;
 using Hazelcast.IO;
 
-namespace Hazelcast.Client.Spi
+#pragma warning disable CS1591
+ namespace Hazelcast.Client.Spi
 {
+    /// <summary>
+    /// Marker interface for exceptions to indicate that an operation can be retried. 
+    /// E.g. a map.get send to a machine where the partition has just moved to another machine.
+    /// </summary>
     [Serializable]
     public class RetryableHazelcastException : HazelcastException
     {
+        /// <inheritdoc />
         public RetryableHazelcastException()
         {
         }
 
+        /// <inheritdoc />
         public RetryableHazelcastException(string message) : base(message)
         {
         }
     }
 
+    /// <summary>
+    /// A <see cref="RetryableHazelcastException"/> that indicates operation is send to a machine that isn't member of the cluster
+    /// </summary>
     [Serializable]
     public class TargetNotMemberException : RetryableHazelcastException
     {
+        /// <inheritdoc />
         public TargetNotMemberException(string message) : base(message)
         {
         }
     }
 
+    /// <summary>
+    /// Thrown when a client invocation is failed because related target is disconnected and
+    /// whether the invocation is run or not, is indeterminate
```

Comment:
...failed because the related target is disconnected, and whether the invocation runs or not is indeterminate.